external/mspack/rules.mk

```diff
@@ -0,0 +1,78 @@
+# xoreos - A reimplementation of BioWare's Aurora engine
+#
+# xoreos is the legal property of its developers, whose names
+# can be found in the AUTHORS file distributed with this source
+# distribution.
+#
+# xoreos is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# xoreos is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+
+# libmspack (<https://www.cabextract.org.uk/libmspack/>)
+
+noinst_LTLIBRARIES += external/mspack/libmspack.la
+external_mspack_libmspack_la_SOURCES =
+
+external_mspack_libmspack_la_CFLAGS = -Iexternal/mspack
```

Comment:
That needs to be `-I$(srcdir)/external/mspack`, otherwise out-of-tree builds fail.